Makefile

```diff
@@ -28,7 +29,7 @@ build:
 	GOOS=$(GOOS) go build ./...
 
 all: check build binary
-check: assert-fmt lint vet
```

Comment:
I don't necessarily have an opinion here, just curious what made you decide to remove `assert-fmt` and `vet` from the checks?

Reply:
These are all handled by `golangci-lint` (could have been a different PR though).